neqo-transport/src/lib.rs

```diff
@@ -26,7 +26,7 @@ mod path;
 mod qlog;
 mod quic_datagrams;
 mod recovery;
-mod recv_stream;
+pub mod recv_stream;
```

Comment:
I would prefer that you re-export the stats part.  That is,

```rust
mod recv_stream;
// ...
pub use recv_stream::RecvStreamStats;
```

There is stuff in the module that we don't need to make public, such as all the RecvStream stuff.

I know that we did that for send_stream, but that was probably also a mistake.